packages/lab/src/tabs-next/TabsNext.tsx

```diff
@@ -0,0 +1,74 @@
+import { Children, cloneElement, isValidElement, ReactNode } from "react";
+import { StackLayout, useControlled, useId } from "@salt-ds/core";
+import { TabstripNext, TabstripNextProps } from "./TabstripNext";
+import { TabNext } from "./TabNext";
+import { TabPanelNext } from "./TabPanelNext";
+import { TabElement } from "./TabsNextTypes";
+
+export type TabsNextProps = Omit<TabstripNextProps, "defaultActiveTabIndex"> & {
```

Comment:
So props are different? `defaultActiveTabIndex` vs `defaultSelectedTab`

Reply:
Whoops, that is a leftover from the previous version. Will remove